platform/queue/queue.go

```diff
@@ -75,6 +76,7 @@ func (db *Store) nextCommand(ctx context.Context, resp mdm.Response) (*Command,
 	case "Acknowledged":
 		// move to completed, send next
 		x, a := cut(dc.Commands, resp.CommandUUID)
+		x.Acknowledged = time.Now()
```

Comment:
Thoughts on renaming this to `AcknowledgedAt` here and in device_command.go?  I'm thinking it could make someone reading this at a call site understand it's a timestamp rather than a boolean.  (also other time properties are suffixed with `At`)

Reply:
That's not a bad idea, but it should be a separate issue/PR. please file it.